GNUmakefile

```diff
@@ -258,6 +258,13 @@ hclfmt:
 	@find . -path ./terraform -prune -o -name 'upstart.nomad' -prune -o \( -name '*.nomad' -o -name '*.hcl' \) -exec \
 sh -c 'hclfmt -w {} || echo in path {}' ';'
 
+.PHONY: hcllint
+hcllint:
+	@find . -path ./terraform -prune -o -name 'upstart.nomad' \
+		-prune -o \( -name '*.nomad' -o -name '*.hcl' \) \
+		-exec bash -c "diff {} <(hclfmt {})" ';' \
+		| tee /dev/stderr | wc -l | head -1 | grep -q 0
```

Comment:
Oops, just realized this bit is wrong `grep -q 0`. Need to make sure that it's _only_ "0"